package.json

```diff
@@ -22,12 +24,15 @@
     "url": "git@github.com:bjrmatos/jsreport-jade.git"
   },
   "dependencies": {
+    "in-publish": "2.0.0",
```

Comment:
@pofider any special reason to have `in-publish` in `dependencies` and not in `devDependencies`?